### ISSUE TITLE
Beta UI: screen reader cannot handle the drop-down text modules.

### ISSUE THREAD

**Issue by klml** [zammad/zammad#6162]
### Used Zammad Version

7.0.1-df77241a.docker

### Environment

Openshift

### Actual behaviour

in BetaUI the screen reader (both Jaws and NVDA) cannot handle the drop-down list for text modules. This means that the screen reader does not recognize the drop-down list and therefore does not read it aloud.

<img width="338" height="364" alt="Image" src="https://github.com/user-attachments/assets/ec70da53-62e3-4856-bbbd-a2b410624ab4" />

### Expected behaviour

 Screen readers should be able to use the text modules in some way, either via the inline dropdown or through an alternative method

### Steps to reproduce the behaviour

Use a screenreader

### Support Ticket

Ticket#10226081

### I'm sure this is a bug and no feature request or a general question.

yes
